solhint.config.js

```diff
@@ -1,16 +1,22 @@
 const customRules = require('./scripts/solhint-custom');
 
 const rules = [
-  'no-unused-vars',
+  'avoid-tx-origin',
   'const-name-snakecase',
   'contract-name-camelcase',
   'event-name-camelcase',
+  'explicit-types',
   'func-name-mixedcase',
   'func-param-name-mixedcase',
-  'modifier-name-mixedcase',
-  'var-name-mixedcase',
   'imports-on-top',
+  'modifier-name-mixedcase',
+  'no-console',
```

Comment:
[Similar](https://protofire.github.io/solhint/docs/rules/best-practises/no-console.html#version), but to 3.4.0.